homeworks/evgeny.tolchinsky_ryvik13/2-css-ryvik13/games.css

```diff
@@ -0,0 +1,282 @@
+  grid-template-rows: 1fr 1fr;
+  align-items: center;
+  justify-content: start;
+  margin-top: 140px;
+}
+
+.games-content {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 100px;
+}
+
+.text-heading {
+  margin-top: 0;
+  color: #221F1F;
+  font-size: 48px;
+  line-height: 32px;
+  font-weight: 700;
+}
+
+.games-cards {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
+  gap: 25px;
+}
+
+.Overwatch2 {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  height: 275px;
+  width: 260px;
+  background-color: #ECECEC;
+  padding: 12px 12px 22px;
+  border-radius: 12px;
+}
+
+.ow2-general {
+  display: flex;
+  flex-direction: row;
+  gap: 12px;
+}
+
+.ow2-general-img img {
+  height: 90px;
+  width: 90px;
+  border-radius: 9px;
+  object-fit: cover;
+}
+
+.ow2-general-desc {
+  display: flex;
+  flex-direction: column;
+  gap: 12px;
+  height: 52px;
+  width: 149px;
+}
+
+.ow2-general-desc h3 {
+  font-size: 18px;
+  margin: 0;
+}
+
+.ow2-general-desc p {
+  font-size: 14px;
+  margin: 0;
+}
+
+.ow2-description {
+  font-size: 14px;
+  line-height: 21px;
+  text-align: start;
+  gap: 16px;
+  padding: 0;
+  margin: 0;
+}
+
+.ow2-description li b {
+  font-weight: 700;
+}
+
+.ow2-description li {
+  font-weight: 400;
+  list-style-type: none;
+  margin-top: 14px;
+  margin-bottom: 14px;
+}
+
+.ow2-description li:last-child {
+  margin-bottom: 0;
+}
+
+/*footer*/
+
+.footer {
+  width: 100%;
+  background-color: #1C2321;
+}
+
+.footer-container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 160px;
+}
+
+.footer-container {
+  width: 100%;
+}
+
+.footer-box-content {
+  width: 90%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+}
+
+.social-icon {
+  height: 42px;
+  width: 192px;
+  padding: 0;
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  align-items: flex-start;
+  gap: 33px;
+  position: relative;
+  list-style-type: none;
+}
+
+.icon-fb {
+  background-color: #FFF;
+  height: 42px;
+  width: 42px;
+  border-radius: 50%;
+}
+
+.icon-fb img {
+  background-color: #FFF;
+  height: 20px;
+  width: 20px;
+  width: 20px;
+  height: 20px;
+  left: 11px;
+  top: 11px;
+  position: absolute;
+}
+
+.icon-link {
+  background-color: #FFF;
+  height: 42px;
+  width: 42px;
+  border-radius: 50%;
+}
+
+.icon-link img {
+  background-color: #FFF;
+  height: 20px;
+  width: 20px;
+  width: 20px;
+  height: 20px;
+  left: 86px;
+  top: 11px;
+  width: 20px;
+  height: 20px;
+  position: absolute;
+}
+
+.icon-inst {
+  background-color: #FFF;
+  height: 42px;
+  width: 42px;
+  border-radius: 50%;
+}
+
+.icon-inst img {
+  background-color: #FFF;
+  height: 20px;
+  width: 20.000892639160156px;
+  width: 20.000892639160156px;
+  height: 20px;
+  left: 161px;
+  top: 11px;
+  width: 20.000892639160156px;
+  height: 20px;
+  position: absolute;
```

Comment:
I think you can insert an image inside of the link without absolute. Just add proper size